app/server.py

```
@@ -272,3 +293,263 @@ async def play_audio(file_path: str) -> str:
+        "model": model,
+        "prompt": prompt_payload,
+        "output": output_payload,
+    }
+
+    safe_text = re.sub(r"\s+", "", text[:10])
+    output_path = (
+        OUTPUT_DIR
+        / f"{datetime.now().strftime('%Y%m%d-%H%M%S')}_{voice_id}_{safe_text}_stream.{audio_format}"
+    )
+
+    async with httpx.AsyncClient(timeout=None) as client:
```

Ruff (0.15.12)

[error] Probable use of `httpx` call with timeout set to `None`

(S113)